dessia_common/workflow/blocks.py

```diff
@@ -417,6 +458,18 @@ def _docstring(self):
                     docstring[input_] = block_docstring[input_]
         return docstring
 
+    def _to_script(self) -> ToScriptElement:
+        PREFIX = 'sub_'
```

Comment:
If this is a constant, it should be placed at module level, and in this case, not passed as an argument.

But I guess what you want to achieve is to customize its value depending on the subworkflow we are working on. What happens if there are several subworkflows ?

Reply:
Alright, in this case, I guess, `prefix` is sufficient. it feels weird pythonically speaking to have a constant defined in a method, even if it means locally in the method scope. But I am a not technically against it, it just triggered something weird that got me worried. Like setting the name all caps would make the value instantly available at module level.

But as your exemple shows, it works perfectly

Reply:
I set it back to lowercase `prefix`